src/app/terminology/terminology-details/terminology-details.component.ts

```diff
@@ -53,46 +57,26 @@ import { ModalDialogStatus } from '@mdm/constants/modal-dialog-status';
   styleUrls: ['./terminology-details.component.sass']
 })
 export class TerminologyDetailsComponent implements OnInit, OnDestroy {
-  @Input() mcTerminology: any;
-  @Input() hideEditButton: boolean;
-  @Output() afterSave = new EventEmitter<any>();
-  @Output() openEditFormChanged = new EventEmitter<any>();
-  @ViewChild('aLink', { static: false }) aLink: ElementRef;
+  @Input() mcTerminology: ModelableDetail & Modelable & Branchable & Versionable & SecurableModel;
```

Comment:
Doesn't `mdm-resources` have a `TerminologyDetail` type to use? Not sure why this is re-combining traits that `TerminologyDetail` should already have.